### PR TITLE
stable/rabbitmq-ha: automount service account API token

### PR DESCRIPTION
**Pull request by barryib**
#### What this PR does / why we need it:
Automount API credentials for a service account by default. This is needed for authz during the service discovery.

I also open a PR in the discovery plugin repo https://github.com/rabbitmq/rabbitmq-peer-discovery-k8s/pull/54

cc @steven-sheehy 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by barryib** [comment]
cc @sameersbn

File: stable/rabbitmq-ha/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: rabbitmq-ha
 apiVersion: v1
 appVersion: 3.7.19
-version: 1.34.0
+version: 1.34.1
```
Comment (steven-sheehy) [inline stable/rabbitmq-ha/Chart.yaml]: `1.35.0` for new features. Other than that it looks good.
/assign

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

File: stable/rabbitmq-ha/Chart.yaml (hunk 1 above)
Comment (barryib) [reply stable/rabbitmq-ha/Chart.yaml]: bumped to 1.35.0 in https://github.com/helm/charts/pull/17987/commits/29b1dcbd2123dcb06df248c15024c9c4c0569e7b

**Comment by barryib** [comment]
@steven-sheehy Any chance to see this merged ?

**Comment by steven-sheehy** [comment]
Sorry, please fix the merge conflict and I can approve.

**Comment by barryib** [comment]
@steven-sheehy conflict resolved.

File: stable/rabbitmq-ha/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 name: rabbitmq-ha
 apiVersion: v1
 appVersion: 3.8.0
-version: 1.35.0
+version: 1.35.1
```
Comment (steven-sheehy) [inline stable/rabbitmq-ha/Chart.yaml]: Again, minor increase please

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

**Comment by steven-sheehy** [comment]
Need to sign off your last commit

**Comment by barryib** [comment]
@steven-sheehy done !

**Review by steven-sheehy — APPROVED** [review]
/lgtm
Thanks for your patience

**Pull request merged**